code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -38,6 +38,19 @@
 	H.adjustToxLoss(5)
 	return TRUE
 
+/datum/ritual/cruciform/base/flare
+	name = "Trickling Light"
+	phrase = "Absumet heres Caecuba dignior sit illuminatus."
```

Comment:
```suggestion
	phrase = "Nun lucerna pedi meo verbum tuum et lux semitae meae."
```
Following the trend of Absolutism having literal bible quotes, psalm 119:105 is the most fitting here. And also more grammatically correct.